src/test/e2e/api/openapi/__snapshots__/openapi.e2e.test.ts.snap

```diff
@@ -3644,26 +3644,7 @@ Stats are divided into current and previous **windows**.
         ],
         "type": "object",
       },
-      "tagsBulkAddSchema": {
-        "additionalProperties": false,
-        "properties": {
-          "features": {
-            "items": {
-              "minLength": 1,
-              "type": "string",
-            },
-            "type": "array",
-          },
-          "tags": {
-            "$ref": "#/components/schemas/updateTagsSchema",
-          },
-        },
-        "required": [
-          "features",
-          "tags",
-        ],
-        "type": "object",
-      },
+      "tagsBulkAddSchema": {},
```

Comment:
is it a mistake? I wouldn't expect his change in this PR

Reply:
Honestly - this schema appeared after merging main in my branch - but it was missing from an index file and a test was failing and that is why it is added here

Reply:
here = the index.ts file not the snapshot